src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -157,3 +158,13 @@ const natureOfControlTypeMap = new Map<string, NatureOfControlType>([
   [natureOfControl.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM, NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL]
 ]);
 
+export const mapBoPrivateAddress = (boPrivateData: BeneficialOwnersPrivateData, ch_reference: string, hasOffice: boolean) => {
```

Comment:
Is this supposed to be for hasOfficeAddress (principalAddress?). Maybe rename to hasOfficeAddress?

Reply:
@AndreKramer7654 👀